integration/test/performancetest/performance_test.go

```diff
@@ -35,11 +35,10 @@ type LogInfo struct {
 func TestPerformance(t *testing.T) {
 	//get number of logs for test from github action
 	//@TODO
-	//logNum, err := strconv.Atoi(os.Getenv(testLogNum)) //requires a commit from Okan that updates the workflow file so the log tests will run concurrently
-	// if err != nil {
-	// 	t.Fatalf("Error: cannot convert test log number to integer, %v", err)
-	// }
-	logNum := 10 //THIS IS TEMPORARY SO CODE RUNS
+	logNum, err := strconv.Atoi(os.Getenv(testLogNum)) //requires a commit from Okan that updates the workflow file so the log tests will run concurrently
```

Comment:
Still standby opinions that these variables should be passed as flag instead of env

Reply:
There isn't enough benefit for making that change.